core/src/main/java/org/mskcc/cbio/oncokb/apiModels/MutationEffectResp.java

```diff
@@ -2,12 +2,22 @@
 
 import java.io.Serializable;
 
+import io.swagger.annotations.ApiModelProperty;
+
 /**
  * Created by Hongxin Zhang on 2/21/18.
  */
 public class MutationEffectResp implements Serializable {
+    @ApiModelProperty(value = "Indicates the effect of the mutation on the gene. Defaulted to \"\"", 
+    allowableValues = "Gain-of-function, Inconclusive, Loss-of-function, Likely Loss-of-function, " +
+    "Likely Gain-of-function, Neutral, Unknown, Likely Switch-of-function, " +
+    "Switch-of-function, Likely Neutral")
     String knownEffect = "";
+
+    @ApiModelProperty(value = "A brief overview of the biological and oncogenic effect of the VPS. Defaulted to \"\"")
```

Comment:
What's VPS

Reply:
Variant of Possible Significance - I copied directly from SOP, but I can spell it out

Reply:
Actually, I can just simplify to "variant" instead